Forge.TreeWalker/src/ExpressionExecutor.cs

```diff
@@ -23,19 +25,36 @@ namespace Microsoft.Forge.TreeWalker
     public class ExpressionExecutor
     {
         /// <summary>
-        /// List of external type dependencies needed to compile expressions.
+        /// Code that Forge uses to prime the parentScript on initialization.
+        /// Evaluating some code moves roughly 600ms from the first expression to initializing the parentScript.
         /// </summary>
-        private List<Type> dependencies;
+        public static string ParentScriptCode = "(1+1).ToString()";
```

Comment:
>"(1+1).ToString()" [](start = 48, length = 18)

nit: consider to add the comment why to add the NO-OP statement.

Reply:
I've outlined the various feature improvements on the new fields/properties. I describe more about the parentScript below. I didn't do deep investigation into how or why adding code here works. I just noticed it in the testing. It didn't matter too much what the code I put here was. I seemed to get slightly better times if the logic here matched the first executed expression logic, such as doing arithmetic. But since the improvements were small and it entirely depends on the unique ForgeTree that each application uses, I don't think its worth any more investment to investigate and improve.

---
In reply to: [470868792](https://github.com/microsoft/Forge/pull/25#discussion_r470868792) [](ancestors = 470868792)